src/server/game/Spells/SpellScript.h

```diff
@@ -142,6 +142,9 @@ class TC_GAME_API _SpellScript
         // Function called when script is destroyed
         // use for: deallocating memory allocated by script
         virtual void Unload() { }
+        // Function called when spell effects with JumpArrivalAction are executed
+        // use for: create your own lambda actions which are called upon finalizing the movement
```

Comment:
Use for: creating your own... :warning:

Reply:
that was just the format that the other methods were using, i think the "use for" is more like a lable, so i reused it

Reply:
Right above virtual void Unload() { } you can see the option used is "use for: deallocat**ing**" :eyes: